deploy/olm-catalog/akka-cluster-operator/0.2.1/akka-cluster-operator-certified.v0.2.1.clusterserviceversion.yaml

```diff
@@ -170,7 +170,7 @@ spec:
                       - name: OPERATOR_NAME
                         value: akka-cluster-operator
                       - name: RELATED_IMAGE_CONTROLLER
-                        value: registry.connect.redhat.com/lightbend/akkacluster-operator:v0.2.0   
+                        value: registry.connect.redhat.com/lightbend/akka-cluster-operator-certified:0.2.0
```

Comment:
should this be version 0.2.1?

Reply:
Actually, no. If you look at the [images in the project](https://connect.redhat.com/project/2073491/view), you'll see a 0.2.0, 0.2.2 and 1.0.1. Of course you could publish an 0.2.0 tag but there is not currently such a tag in the certified registry.